cc-mt-service/src/routes/mtServiceRoutes.ts

```diff
@@ -0,0 +1,13 @@
+// Mt service routes
+import { Router, Request, Response } from "express";
+import {postAdvert} from '../handleData';
+
+const mtRouter = Router();
+
+mtRouter.post('/advert',async (req:Request, res:Response) => {
+    const response = await postAdvert();
+    res.send(response);
```

Comment:
This needs some error handling around, take a look at cc-pk-service/src/routes/pkServiceRoutes.ts for example